hpqtypes-extras.cabal

```diff
@@ -1,5 +1,5 @@
 name:                hpqtypes-extras
-version:             1.3.1.1
+version:             1.4
```

Comment:
Please use a 4-component version number (i.e. `1.4.0.0`) for consistency.

Reply:
Also please update the changelog.

Reply:
gotcha